### DISCUSSION TITLE
Adding Custom Phonetics

### DISCUSSION THREAD

**Discussion by lothsun** [antihax/RadioNotifications#3] [General]
How would one go about adding custom phonetics? I know it would just be adding wavs but do you have a demo script for overriding/adding a new voice/audio files?

**Comment by antihax** [comment]
On load, it checks for soundsets and loads in order until no more remain.
https://github.com/antihax/RadioNotifications/blob/main/scripts/game/radionotificationsettings.c#L111

You should be able to make a mod which adds more soundsets in continuation of the existing configuration:

https://github.com/antihax/RadioNotifications/blob/main/config.cpp#L4625

**Reply by lothsun** [reply]
So if I did `RadioNotification_Voice10_Phonetic0` through `RadioNotification_Voice10_Phonetic100` in config.cpp extending the `RadioNotification_SoundSet` it would recognize those as voice 10 and then I could reference phonetic 73 if i wanted to?

**Reply by lothsun** [reply]
and what's the difference between grid reference 6n and 4n? Would using that use voice 10 as well as long as i have the same starting phonetics order as your other voices?

**Reply by Kaufman70** [reply]
Did you manage to create a extension for the messages, if so could you share the file.
